app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -79,6 +79,8 @@ interface Cta {
 }
 
 interface OnboardingDaxCta {
+    val markAsReadOnShow: Boolean
```

Comment:
You can use `val markAsReadOnShow: Boolean get() = false` instead, then you don’t have to set it to false every time.